cmd/agent/agent.go

```diff
@@ -0,0 +1,51 @@
+package agent
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/mitchellh/go-homedir"
+	"github.com/spf13/viper"
+
+	"github.com/SUSE/console-for-sap-applications/cmd/webapp"
+)
+
+var cfgFile string
+
+var cmdRegister = &cobra.Command{
+	Use:   "register [some arguments]",
+	Short: "Register the agent in the system",
+	Long: `Register the agent in the
+system`,
+	Args: cobra.MinimumNArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		fmt.Println("You have registered your agent with arguments: " + strings.Join(args, " "))
+	},
+}
+
+var cmdStart = &cobra.Command{
+	Use:   "start [with arguments]",
+	Short: "Start the agent",
+	Long: `Start the
+agent`,
+	Args: cobra.MinimumNArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		fmt.Println("You have started the agent with the arguments: " + strings.Join(args, " "))
+	},
+}
+
+// Execute adds all child commands to the root command and sets flags appropriately.
+// This is called by main.main(). It only needs to happen once to the rootCmd.
+func Execute() {
+	cobra.CheckErr(cmdRegister.Execute())
+	cobra.CheckErr(cmdStart.Execute())
+}
+
+func init() {
+	cobra.OnInitialize(initConfig)
+	rootCmd.AddCommand(cmdRegister)
+	rootCmd.AddCommand(cmdStart)
+}
+
```

Comment:
this part we don't need to put it here.

we need to put and adapt it here: https://github.com/SUSE/console-for-sap-applications/blob/main/cmd/root.go#L34

Reply:
the init part is placed on `root.go` file. in this we just need to implement the cobra part

Reply:
I've removed the init function completely.